### PR TITLE
Add a workflow that runs Emscripten unit tests with the release package

### PR DESCRIPTION
**Pull request by Blackhex**
Should be ready except of `pip install` issue with libxml on the VM runner. It would be worth trying if this cannot be resolved by using stable Build Tools.

### CODE REVIEW

File: .github/workflows/test-package.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        working-directory: ${{ github.workspace }}
+        run: New-Item -ItemType Directory -Force -Path test-package
+
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          path: test-package
+
+  install-package:
+    needs: bootstrap-workflow
```
Comment (eukarpov) [inline .github/workflows/test-package.yml]: Now only one runner is used, however jobs should be independent

Comment (Blackhex) [reply .github/workflows/test-package.yml]: Do I understand correctly that you are suggesting that each job should clone it's own version of emsdk repository to a different directory?

Comment (eukarpov) [reply .github/workflows/test-package.yml]: Correct, maybe it makes sense to join them into one job.

Comment (Blackhex) [reply .github/workflows/test-package.yml]: Ignore my previous comment, I thought you are commenting on a different workflow.

Anyway, I've refactored the wrokflow to a single job.

**Comment by Blackhex** [comment]
Changed to run on default runner. Now the Python dependencies installs fine and the hello world test passes  - ready for review.

**Review by eukarpov — APPROVED** [review]

**Pull request merged**